editor/animation_track_editor.cpp

```diff
@@ -7297,6 +7305,19 @@ AnimationTrackEditor::AnimationTrackEditor() {
 
 	bottom_hb->add_child(bezier_edit_icon);
 
+	ED_SHORTCUT("animation_editor/hide_function_name", TTR("Hide function name"));
+
+	function_name_toggler = memnew(Button);
+	function_name_toggler->set_flat(true);
+	function_name_toggler->set_disabled(false); // TODO: Enable it only when there is a method call.
+	function_name_toggler->connect(SceneStringName(pressed), callable_mp(this, &AnimationTrackEditor::_toggle_function_name));
+	function_name_toggler->set_shortcut(ED_GET_SHORTCUT("animation_editor/hide_function_name"));
+	function_name_toggler->set_toggle_mode(true);
+	function_name_toggler->set_shortcut_in_tooltip(false);
+	function_name_toggler->set_tooltip_text(TTR("Hide names of functions in tracks."));
+
+	bottom_hb->add_child(function_name_toggler);
+
```

Comment:
```suggestion
	ED_SHORTCUT("animation_editor/toggle_function_name", TTR("Toggle function name"));

	function_name_toggler = memnew(Button);
	function_name_toggler->set_flat(true);
	function_name_toggler->set_disabled(false); // TODO: Enable it only when there is a method call.
	function_name_toggler->connect(SceneStringName(pressed), callable_mp(this, &AnimationTrackEditor::_toggle_function_name));
	function_name_toggler->set_shortcut(ED_GET_SHORTCUT("animation_editor/toggle_function_name"));
	function_name_toggler->set_toggle_mode(true);
	function_name_toggler->set_shortcut_in_tooltip(false);
	function_name_toggler->set_tooltip_text(TTR("Toggle function names on the track editor."));

	bottom_hb->add_child(function_name_toggler);

```
The current state is indicated by the appearance of the icon, and considering the other options, the explanation “Toggle” is more appropriate than “Show/Hide”.